kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/JdbcColumn.java

```diff
@@ -178,6 +178,7 @@ public static int convertTTypeIdToSqlType(TTypeId type) throws SQLException {
       case INTERVAL_YEAR_MONTH_TYPE:
       case INTERVAL_DAY_TIME_TYPE:
       case UNION_TYPE:
+      case USER_DEFINED_TYPE:
```

Comment:
please implement getColumnTypeName

Reply:
And other APIs

Reply:
thanks, addressed

Reply:
Seems other apis all leverage `convertTTypeIdToSqlType`